dash_draggable/package.json

```diff
@@ -15,6 +15,7 @@
     "author": "Xing Han Lu xinghan@plot.ly",
     "license": "MIT",
     "dependencies": {
+        "acorn": "^6.1.0",
```

Comment:
Could you remove this please? that way I won't need to make a new PR :)

Reply:
Good Point